pwndbg/config.py

```diff
@@ -113,19 +132,49 @@ def native_default(self):
     def is_changed(self):
         return self.value != self.default
 
+    def __setattr__(self, name, value):
+        new_name = member_remap.get(name, name)
+        new_name = str(new_name) # Python2 only accept str type as key
+        return super(Parameter, self).__setattr__(new_name, value)
+
+    def __getattribute__(self, name):
+        new_name = member_remap.get(name, name)
+        new_name = str(new_name) # Python2 only accept str type as key
+        return super(Parameter, self).__getattribute__(new_name)
+
     def get_set_string(self):
+        value = self.raw_value
+
+        # For string value, convert utf8 byte string to unicode.
+        if isinstance(value, six.binary_type):
+            value = codecs.decode(value, 'utf-8')
+
+        # Remove surrounded ' and " characters
```

Comment:
The original code implement this and use `replace` to do that. I think it is because `gdb.Parametor` will keep all characters. That is, `set color 'bold'` will get a string `'bold'` rather than `bold`. I guess people this to make sure  `set color 'bold'` works the same as `set color bold` since programmer often like to add quotes on string literal.

> How do I intentionally force quotes into a value?
Currently we can't put quote at the beginning and ending. Before my commit, your implement use `replace` to do that. It will remove all the quotes in the string, but I don't see any complain about that. Use `strip` will much better than before.

Reply:
@anthraxx @disconnect3d 
BTW, I saw the PR #409 reply on the quotes. However, the implementation of `dev` `HEAD` will remove all quotes by `replace`. 
https://github.com/pwndbg/pwndbg/blob/103b1def610957e2c74ab616b15a0513a1286cd6/pwndbg/config.py#L120

Reply:
It seems it will be set after `trigger()` so the checker will get the raw value. So PR #409 works fine and will be broken by this PR.

However, the in-consistence between trigger and final value will be a problem? For example, the trigger function does not expect the quote will be removed.

Can we re-design what we expect to do before/after the trigger function?

Reply:
@cebrusfs I like the idea of removing (well replacing) the quotes.

I don't really understand why would it break #409? Anyway, if so, it shouldn't :P

Reply:
@disconnect3d 
The original implementation has these feature. I just use `strip` to remove quotes instead of use `replace` to avoid break some string that contains quotes in the middle. Ex. `"me'ow"` should be `me'ow` (after), not `meow` (before).

I said it will break since the quote removing happened after calling wrapper function (`Trigger`) in the original version.  That is the value `'meow'` will still be `'meow'` when the wrapper fetches the value from `self.value` and after that it will be striped and becomes `meow`.

However, my implementation removes the quotes at the first, so the wrapper will get `meow` in these case. So `""` and `''` case will not happend in your code `config_context_sections.value.lower() in ('""', "''", 'none', 'empty'):`.

One way to fix that is moving the quote replacing step after calling the wrapper function (keep original order).

However, this preprocess happens after calling the wrapper functions is weird since the needed of removing quote is caused by gdb argument parser. In python environment (wrapper function), I think it is better to not to have any extra quote striping.